pkg/rebuild/maven/infer.go

```diff
@@ -157,6 +158,22 @@ func getJarJDK(ctx context.Context, name, version string, mux rebuild.RegistryMu
 	if err != nil {
 		return "", errors.Wrap(err, "unzipping jar file")
 	}
+	jdk, err := inferJDKFromManifest(zipReader)
+	if err != nil {
+		return "", err
+	}
+	if jdk != "" {
+		return jdk, nil
+	}
+	jdkInt, err := inferJDKFromBytecode(zipReader)
+	if err != nil {
+		return "", err
```

Comment:
nit: i think we want to wrap this error and the one above it to give context about where the error came from. we tend to phrase these wraps in the present progressive tense e.g. "doing X" so the error ends up formatted "doing X: no .class files found"